conftest.py

```diff
@@ -7,8 +7,9 @@
 from array_api_tests import _array_module as xp
 from array_api_tests._array_module import _UndefinedStub
 
-settings.register_profile("xp_default", deadline=800)
+from reporting import pytest_metadata, add_extra_json_metadata # noqa
```

Comment:
Could you add a small comment explaining what's being imported here and why? I assume it's doing some magic when imported.

Reply:
Yes, the names have to be defined here so that pytest will see them as fixtures. I decided to keep the reporting in a separate file to keep things clean, since it's already a good bit of code and could potentially be more if we decide to add more stuff to the report in the future.